### PR TITLE
Add ES6 support with Babel

### PR DESCRIPTION
**Pull request by vincemtnz**
Figure it might be useful after all? List of changes:
- move `js/index.js` to `src/index.js`
- configure rollup to read `src/index.js` and output to `dist/bundle.js`. updated index.html to reference that script instead of `js/index.js`.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +48,6 @@
 
 So you can see how using the <code>bind()</code> method allows us to control the value of <code>this</code> within the
 <code>appendStringToThisGreeting()</code> function.
-</body>
```
Comment (mcalthrop) [inline index.html:50]: Did I really put the `</body>` tag here?
How did I manage that? 😸

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "js-sandbox",
+  "version": "0.1.0",
+  "description": "Sandbox for JavaScript testing",
+  "main": "lib/index.js",
```
Comment (mcalthrop) [inline package.json]: Is this the main file?

**Comment by mcalthrop** [comment]
I'll need to add a section to the `README` to describe how to set it up, but that will be fine.

File: package.json (hunk 2 above)
Comment (vincemtnz) [inline package.json]: ah i made a mistake, should've ommited that field.

File: index.html [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 <head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>JS Sandbox</title>
-    <script src="js/index.js"></script>
     <link rel="stylesheet" type="text/css" href="css/index.css"/>
 </head>
 
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+</head>
```
Comment (mcalthrop) [inline index.html]: Something's not right here... the CSS stylesheet is included twice, and the `</head>` is there twice as well.

**Pull request merged**